pkg/morph/client/reputation/wrapper/get.go

```diff
@@ -0,0 +1,95 @@
+package wrapper
+
+import (
+	"github.com/nspcc-dev/neofs-api-go/pkg/reputation"
+	reputationClient "github.com/nspcc-dev/neofs-node/pkg/morph/client/reputation"
+	"github.com/pkg/errors"
+)
+
+type (
+	// GetArgs groups the arguments of "get reputation value" test invocation.
+	GetArgs struct {
+		epoch  uint64
+		peerID reputation.PeerID
+	}
+
+	// GetByIDArgs groups the arguments of "get reputation value by
+	// reputation id" test invocation.
+	GetByIDArgs struct {
+		id ReputationID
+	}
+
+	// ListByEpochResult groups the result of "get reputation value" and
+	// "get reputation value by reputation id" test invocations.
+	GetResult struct {
+		reputations []reputation.GlobalTrust
+	}
+)
+
+// SetEpoch sets epoch of expected reputation value.
+func (g *GetArgs) SetEpoch(v uint64) {
+	g.epoch = v
+}
+
+// SetPeerID sets peer id of expected reputation value.
+func (g *GetArgs) SetPeerID(v reputation.PeerID) {
+	g.peerID = v
+}
+
+// SetID sets id of expected reputation value in reputation contract.
+func (g *GetByIDArgs) SetID(v ReputationID) {
+	g.id = v
+}
+
+// Reputations returns slice of reputation values.
+func (g GetResult) Reputations() []reputation.GlobalTrust {
+	return g.reputations
+}
+
+// Get invokes the call of "get reputation value" method of reputation contract.
+func (w *ClientWrapper) Get(v GetArgs) (*GetResult, error) {
+	args := reputationClient.GetArgs{}
+	args.SetEpoch(v.epoch)
+	args.SetPeerID(v.peerID.ToV2().GetValue())
```

Comment:
Why don't we use `v.peerID.Marshal()` (same in `wrapper.preparePutArgs`)?

Reply:
PeerID value used as a part of the key for storage in reputation contract. Storage key has it's own size limits, so I decided to keep exact value right there without extra meta information of protobuf encoding. This should encourage developers to not change `reputation.PeerID` definition in API.

Reply:
But why contract's storage key imposes requirements to `PeerID` size? It becomes pretty version-dependent then, same as this part of the code.

Reply:
> But why contract's storage key imposes requirements to PeerID size? 

It is NEO limitation of max storage key size (64 byte), so we can't do anything about that. 

Another thing is that bytes of marshaled `PeerID` structure is not PeerID. The same as bytes of marshaled structure of ContainerID is not ContainerID, so it seemed legit to use value in this case.

Reply:
I got it, in the background I will think about how to simultaneously adjust to the restrictions of the contract and avoid using `ToV2()` (apparently some way of getting a binary ID of a fixed size).